python/pyspark/logger/logger.py

```diff
@@ -26,7 +27,7 @@
     "msg STRING, "
     "context map<STRING, STRING>, "
     "exception STRUCT<class STRING, msg STRING, "
-    "stacktrace ARRAY<STRUCT<class STRING, method STRING, file STRING,line STRING>>>,"
+    "stacktrace ARRAY<STRUCT<class STRING, method STRING, file STRING, line STRING>>> ,"
```

Comment:
👍

Reply:
oh, actually, the space should be:
```suggestion
    "stacktrace ARRAY<STRUCT<class STRING, method STRING, file STRING, line STRING>>>, "
```